### PR TITLE
AT_12.02_010 | Freestyle project | Delete created project

### PR DESCRIPTION
**Pull request by Parizat-Ismailova**
https://trello.com/c/9uc73zQs/815-at1202010-freestyle-project-delete-created-project

### CODE REVIEW

**Review by Rom-T — APPROVED** [review]

File: cypress/e2e/freestyleProjectDelete.cy.js [hunk 1]
```diff
@@ -63,4 +63,15 @@ describe('Freestyle project Delete', () => {
     })
     cy.get('#main-panel h1').should('have.text', homePage.loginPageHeader)
   })
+  
+  it('AT_12.02_010 | Freestyle project | Delete created project',()=>{
+    cy.get('tbody tr td a.jenkins-table__link').contains(projects.freestyle.name).realHover({ position: "center" })
+    cy.get('#projectstatus .jenkins-menu-dropdown-chevron').click()
+    cy.get('.first-of-type li:nth-child(5)').should('have.text','Delete Project').click()
```
Comment (Rom-T) [inline cypress/e2e/freestyleProjectDelete.cy.js:70]: Please move `'Delete Project'` in fixtures

File: cypress/e2e/freestyleProjectDelete.cy.js [hunk 2]
```diff
@@ -63,4 +63,15 @@ describe('Freestyle project Delete', () => {
...
+
+    cy.on('window:confirm', (str) => {
+    expect(str).to.equal('Delete Project: are you sure?')
```
Comment (Rom-T) [inline cypress/e2e/freestyleProjectDelete.cy.js:73]: Please move `'Delete Project: are you sure?'` in fixtures

File: cypress/e2e/freestyleProjectDelete.cy.js [hunk 3]
```diff
@@ -63,4 +63,15 @@ describe('Freestyle project Delete', () => {
...
+    })
+    cy.get('#main-panel').contains('Project Freestyle').should('not.exist')
```
Comment (Rom-T) [inline cypress/e2e/freestyleProjectDelete.cy.js:75]: Please move `'Project Freestyle'` in fixtures

**Review by SvetlanaVet — APPROVED** [review]

File: cypress/e2e/freestyleProjectDelete.cy.js (hunk 1 above)
Comment (Parizat-Ismailova) [reply cypress/e2e/freestyleProjectDelete.cy.js:70]: Good morning. Got it, Thanks a lot

File: cypress/e2e/freestyleProjectDelete.cy.js (hunk 2 above)
Comment (Parizat-Ismailova) [reply cypress/e2e/freestyleProjectDelete.cy.js:73]: Got it, thank you

**Review by DariaMas7 — APPROVED** [review]

**Pull request merged**